### ISSUE TITLE
Get users current location

### ISSUE THREAD

**Issue by kerryconvery** [dmsl/anyplace#57]
Hi Guys

In a new activity that I've added to the Anyplace App I'm wanting to know the users currently location within the indoor space, assume they are standing still.  What class or classes do I need to get this information?

I've been looking through the UnifiedNavigationActivity but there is a lot going on so I'm hoping someone can just point me in the right direction.  

I found a method called onNewLocation which seems to be what I need but I can't see where this is called from.   

Basically a user will walk up to something and press a button on the app.  On the button press I want to immediately start querying there current location and then use that position information to query a database.  I may add continuous tracking later but for now pressing a button whenever they want to know what they are standing in front of will suffice.

Thanks

**Comment by constambeys** [comment]
Hello kerryconvery

The main class responsible for indoor tracking is
